pyigm/metallicity/mcmc.py

```diff
@@ -1003,8 +1101,8 @@ def __call__(self, param):
             return lh + lp
 
 
-def mcmc_ions(data,infodata,model,logUconstraint=False, UVB='HM05', nwalkers=500,nsamp=250,threads=12,
-              outsave='emceeout', optim=False, effnhi=True, testing=False):
+def mcmc_ions(data,infodata,model,logUconstraint=False, logUmean=-2.968, logUsigma=0.481, UVB='HM05', nwalkers=400,nsamp=400,threads=12,
+              outsave='emceeout', optim=False, effnhi=True, testing=False, nwalkers_min=400, nsamp_min=400):
```

Comment:
Are these minimum parameters defined in the doc string?
If not, please add them.